app/controllers/sessions_controller.rb

```diff
@@ -4,11 +4,20 @@ class SessionsController < ApplicationController
   def create
     session[:user_token] = auth_hash.fetch(:credentials).fetch(:token)
 
-    redirect_to root_url
+    redirect_to redirect_url
   end
 
   private
 
+  def redirect_url
+    custody_suite_root_url if user_role == "cso"
+  end
+
+  # TODO: This is hacky
+  def user_role
+    @user_role ||= current_user.organisations.map { |o| o["roles"] }.flatten.uniq.first
```

Comment:
I don't think we should consolidate this yet, I think it would be weird to have a `role` and a `roles` method on the user. And there may still be benefit to seeing what other roles the user has.